doc/classes/TabBar.xml

```diff
@@ -214,6 +208,9 @@
 		<member name="tab_close_display_policy" type="int" setter="set_tab_close_display_policy" getter="get_tab_close_display_policy" enum="TabBar.CloseButtonDisplayPolicy" default="0">
 			Sets when the close button will appear on the tabs. See [enum CloseButtonDisplayPolicy] for details.
 		</member>
+		<member name="tab_count" type="int" setter="set_tab_count" getter="get_tab_count" default="0">
+			The number of tabs currently in the list.
```

Comment:
Is the "list" some leftover from another node? It could say "bar" instead of "list" (since it's TabBar 🤔).

Reply:
Yes, leftover from ItemList I guess. Didn't know what to put but I've changed to bar as you suggested